Package.swift

```diff
@@ -233,7 +233,7 @@ let package = Package(
 if ProcessInfo.processInfo.environment["SWIFTPM_LLBUILD_FWK"] == nil {
     if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         package.dependencies += [
-            .package(url: "https://github.com/apple/swift-llbuild.git", .branch("master")),
+            .package(url: "https://github.com/apple/swift-llbuild.git", .branch("release/5.3")),
```

Comment:
this fix seems fine to me, but we need a more systematic way of dealing with this. perhaps an env variable (or similar) that we can use across all version-locked dependencies?

Reply:
How about something like this?
Keeping an explicit default is still important IMO for the 'just add a package in Xcode' case - it's reasonable for libSwiftPM users to expect that to work without extra configuration.

Using same branch for llbuild and TSC should at least eliminate the current particular issue in future, and the constant make it stand out a little as something that needs to be manually changed.
We could also shell out to git and try to guess the corresponding tag, but that can turn to an `update_checkout.py`-style beast, maybe overkill for this.